include/deal.II/matrix_free/matrix_free.h

```diff
@@ -2488,16 +2484,14 @@ MatrixFree<dim, Number, VectorizedArrayType>::n_active_entries_per_cell_batch(
   const unsigned int cell_batch_number) const
 {
   AssertIndexRange(cell_batch_number, task_info.cell_partition_data.back());
-  unsigned int n_components = VectorizedArrayType::n_array_elements;
-  while (
-    n_components > 1 &&
-    cell_level_index[cell_batch_number * VectorizedArrayType::n_array_elements +
-                     n_components - 1] ==
-      cell_level_index[cell_batch_number *
-                         VectorizedArrayType::n_array_elements +
-                       n_components - 2])
+  unsigned int n_components = VectorizedArrayType::size();
```

Comment:
```suggestion
  unsigned int n_simd_filled = VectorizedArrayType::size();
```
and all downstream uses; I must admit that at first I was confused by the use of `n_components` here.